x-pack/plugins/apm/public/hooks/use_fallback_to_transactions_fetcher.tsx

```diff
@@ -33,13 +43,21 @@ export function useFallbackToTransactionsFetcher({ kuery }: { kuery: string }) {
 
   const { data = { fallbackToTransactions: false } } = useFetcher(
     (callApmApi) => {
-      return callApmApi('GET /internal/apm/fallback_to_transactions', {
-        params: {
-          query: { kuery: kueryWithFilters, start, end },
-        },
-      });
+      if (start && end && documentType && rollupInterval) {
```

Comment:
2 things here -
1. I am not 100% sure of this hook needs to be updated. Reason being if you check the backend API call it does, this passed parameter is not used at all.
Also this hook specifically checks for availability of raw transaction events.

2. This makes me wonder do we really need this hook at all now ? the PreferredDataSource hook does provide us the information about document being present in TransactionEvent Index or not. We can actually use that no and get rid of this hook ? @dgieselaar wdyt ?

Reply:
1. fixed here https://github.com/elastic/kibana/pull/170749/commits/34a33688494635e78f0cc44008e938d14b87382e